test_rclcpp/CMakeLists.txt

```diff
@@ -108,6 +108,33 @@ if(BUILD_TESTING)
     endif()
   endmacro()
 
+  macro(custom_launch_n_nodes num_nodes)
+    cmake_parse_arguments(_ARG "" "RMW" "" ${ARGN})
```

Comment:
You don't need to parse arguments for passing this information. You can simply refer to `${rmw_implementation}` within the macro.

Reply:
Ah right, it is a macro.  Fixed in 5a9c83ed1c9fed1b567bcf8a431a0a849e384aca

Reply:
> Ah right, it is a macro.

The same is true for functions...